presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -381,6 +383,39 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
     }
 
     @Override
+    public void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties)
+    {
+        Database database = new Database();
+        database.setName(schemaName);
+
+        HiveSchemaProperties.getLocation(properties).ifPresent(locationUri -> {
+            try {
+                hdfsEnvironment.getFileSystem(session.getUser(), new Path(locationUri));
+            }
+            catch (IOException e) {
+                throw new PrestoException(INVALID_SCHEMA_PROPERTY, "Invalid location URI: " + locationUri, e);
+            }
+            database.setLocationUri(locationUri);
+        });
+
+        PrincipalPrivilegeSet principalPrivilegeSet = buildInitialPrivilegeSet(session.getUser());
```

Comment:
Database only has one permission `ownership` which is set by the ownership field on the database object